web/wp-content/themes/borealis-2021/inc/partials/search-bar.php

```diff
@@ -1,22 +1,69 @@
-<?php 
- 
+<?php
 if (!function_exists('pg_generate_search_bar')) {
-    function pg_generate_search_bar($query) {
+    function pg_generate_search_bar($query, $taxonomies = array())
+    {
+        $taxonomies_array = array();
```

Comment:
@Braedencraig can you move this functionality where you populate the taxonomies to the page-search.php file and then pass it through? As we will need to access these values on the page when we're making a query to WordPress